### ISSUE TITLE
All src packages fail to compile on Kali

### ISSUE THREAD

**Issue by brainthee** [gnuradio/pybombs#184]
Trying to use pybombs on Kali with the target as /usr/local and deb requirements are met but src always reaches 100% and just bombs out. No error message or anything. So far this seems to happen at the first time it tries to compile.

After running the command manually to get the verbose output, I narrowed it down to the issue identified in this stackoverflow thread: http://stackoverflow.com/questions/27401931/how-can-i-install-wxpython-3-0-1-1-in-python-2-7-virtualenv-on-ubuntu-14-10

Basically, a few environment variables are empty and seem to be required. Setting the following seemed to get it going.

`CFLAGS=-Wno-error=format-security CPPFLAGS=-Wno-error=format-security `
